app/views/account/profiles/_form.html.slim

```diff
@@ -1,9 +1,9 @@
 = simple_form_for @user, url: [:account, :profile] do |f|
   = f.input :avatar, label: "Select your profile avatar", wrapper: :upload_file
-  = f.input :first_name, wrapper: :bulma_input, required: true
-  = f.input :last_name, wrapper: :bulma_input, required: true
-  = f.input :job_role, wrapper: :bulma_input
-  = f.input :department, wrapper: :bulma_input
-  = f.input :about, wrapper: :textarea
+  = f.input :first_name, wrapper: :bulma_input_label_left, required: true
```

Comment:
this is not good
I don't want to have 1000 inputs

it's possible to specify class using simple_form

remove this wrapper and do it

Reply:
= f.input :first_name, class: "has-text-left", wrapper: :bulma_input, required: true
 Its doesn`t work. We need to add class inside the wrapper
![image](https://user-images.githubusercontent.com/28151680/48485000-5952db80-e820-11e8-8300-32a8afd29de3.png)
And without any wrapper:
![image](https://user-images.githubusercontent.com/28151680/48485050-80111200-e820-11e8-83ec-30f9632f580b.png)